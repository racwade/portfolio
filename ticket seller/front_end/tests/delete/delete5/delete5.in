login
test_admin
delete
!@#$%
logout