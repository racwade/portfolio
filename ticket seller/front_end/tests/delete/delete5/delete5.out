welcome to the ticket seller system
enter username:
logged in as standard_admin
enter account username:
error: Symbols not accepted
bye